test/README.md

```diff
@@ -424,6 +425,26 @@ All of the functions that act as the scenario API are run in the
 context of `./test/bin/scenario.sh` and can therefore use any functions
 defined there.
 
+#### Scenarios testing between different MicroShift sources
+
+Scenarios utilize following distinct MicroShift sources:
+- `src`: built from source (code in PR)
+- `base`: built from base branch (PR's target branch)
+- `prev-minor`: previous MicroShift minor release
+
+| Starting ref | End ref | Successful upgrade scenario | Failed upgrade scenario |
+|--------------|---------|-----------------------------|-------------------------|
+| `base` | `src` |`el92_base_upgrade-ok.sh` | `el92_base_upgrade-failing.sh` |
+| `prev-minor` | `src` |`el92_prev-minor_upgrade-ok.sh` | `el92_prev-minor_upgrade-failing.sh` |
```

Comment:
prev-minor could be just "prev", unless you think that's not clear.

Reply:
This is a great suggestion, I didn't like the `prev-minor` to much, especially with the `-` it was a little bit unreadable for me